<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>decision_dlg</class>
 <widget class="QWidget" name="decision_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>659</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>依据示范矿区智能决策</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../usecad.qrc">
    <normaloff>:/images/cbm.ico</normaloff>:/images/cbm.ico</iconset>
  </property>
  <widget class="QWidget" name="widget_main" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>591</width>
     <height>621</height>
    </rect>
   </property>
   <widget class="QPushButton" name="graph">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>570</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查 看 煤 层 赋 存 立 体 示 意 图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decision">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>煤层气抽采技术模式决策</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="protect_layer_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>213</y>
      <width>571</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>序号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>煤层编号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>煤层倾角</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>煤层厚度</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>保护类型</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>184</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/arrow.png&quot;/&gt;保护层开采条件&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-2</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/arrow.png&quot;/&gt;煤层渗透率条件&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="permeability_k_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>28</y>
      <width>571</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>序号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>煤层编号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>煤层渗透率</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>煤层渗透率&gt;5mD?</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="coal_population_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>397</y>
      <width>571</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>序号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>煤层编号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>与上覆煤层间距</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>冒落带高度</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>采动影响倍数</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>367</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/arrow.png&quot;/&gt;煤层群条件&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="help1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>1</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;1、中高渗条件&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;所有的煤层渗透率均大于等于&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#aa55ff;&quot;&gt;5mD&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;2、低渗条件&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;煤层渗透率均大于等于&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#aa55ff;&quot;&gt;5mD&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../usecad.qrc">
      <normaloff>:/images/question.png</normaloff>:/images/question.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="help2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>186</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体'; font-size:14pt; color:#000000;&quot;&gt;相对层间距离和煤层倾角是决定保护层保护效果的关键因素，但保护层回采工作面参数对保护也有不可忽视的影响。因此，在反映保护层厚度与层间垂距的相对层间距基础上，综合考虑煤层赋存条件、层间硬岩以及保护层工作面回采参数的影响，构建保护层的工程分类指标——当量相对层间距&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../usecad.qrc">
      <normaloff>:/images/question.png</normaloff>:/images/question.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="help3">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>370</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体'; font-size:14pt; color:#000000;&quot;&gt;1、单一煤层条件&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体'; font-size:12pt; color:#000000;&quot;&gt;（1）煤层数量为&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:12pt; color:#000000;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:12pt; color:#000000;&quot;&gt;层&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体'; font-size:12pt; color:#000000;&quot;&gt;（2）煤层数量大于1层，且任何两层间的层间距与下层面的比值均大于&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:12pt; color:#000000;&quot;&gt;40&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:12pt; color:#000000;&quot;&gt;时，判定为&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;2、&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:14pt; color:#000000;&quot;&gt;煤层群条件&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:12pt; color:#000000;&quot;&gt;（1）&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:12pt; color:#000000;&quot;&gt;近距离煤层群条件&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:12pt; color:#000000;&quot;&gt;任何一个煤层的&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:12pt; color:#000000;&quot;&gt;“&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:12pt; font-weight:600; color:#aa55ff;&quot;&gt;与上覆煤层间距&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:12pt; color:#000000;&quot;&gt;”&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:12pt; color:#000000;&quot;&gt;均&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:12pt; font-weight:600; color:#000000;&quot;&gt;≤&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:12pt; color:#000000;&quot;&gt;“&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:12pt; font-weight:600; color:#aa55ff;&quot;&gt;煤层开采冒落带高度&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:12pt; color:#000000;&quot;&gt;”&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:12pt; color:#000000;&quot;&gt;且&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:12pt; font-weight:600; color:#aa55ff;&quot;&gt;采动影响倍数均≤7.5&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:12pt; color:#000000;&quot;&gt;时，判定为；&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体'; font-size:12pt; color:#000000;&quot;&gt;（2）&lt;/span&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:12pt; color:#000000;&quot;&gt;中远距离煤层群条件&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:12pt; color:#000000;&quot;&gt;不满足（1）条件的煤层&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../usecad.qrc">
      <normaloff>:/images/question.png</normaloff>:/images/question.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_title" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QLabel" name="lab_Ico">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lab_Title">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMenu_Close">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../usecad.qrc"/>
 </resources>
 <connections/>
</ui>
